data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/ReceiverVerticle.java

```diff
@@ -37,20 +37,22 @@
   private HttpServer server;
   private MessageConsumer<Object> messageConsumer;
   private final Function<Vertx, RequestMapper<String, CloudEvent>> requestHandlerFactory;
-  private final Function<Handler<HttpServerRequest>, Handler<HttpServerRequest>>[] handlerDecoratorFactories;
+  private final Function<Handler<HttpServerRequest>, Handler<HttpServerRequest>>[]
+      handlerDecoratorFactories;
```

Comment:
The forced wrap here doesn't make any sense, why should i have the field name on a newline?